packages/clay-management-toolbar/src/ActiveState.tsx

```diff
@@ -0,0 +1,252 @@
+	clearSelectionURL,
+	disabled = false,
+	filterItems,
+	labels = {
+		allSelected: 'All Selected',
+		checkbox: 'Select Items',
+		clearSelection: 'Clear',
+		selectAll: 'Select all',
+		selected: 'Selected',
+		selectedItemsOfTotalItems: '{0} of {1}',
+	},
+	onCheckboxChange = () => {},
+	onClearSelectionClick = () => {},
+	onInfoButtonClick = () => {},
+	onQuickActionClick = () => {},
+	onSelectAllClick = () => {},
+	selectAllURL,
+	selectedItems = 0,
+	showInfoButton = false,
+	showSearch = false,
+	showSelectAllButton = false,
+	sortingOrder,
+	spritemap,
+	supportsBulkActions = false,
+	titlesLang = {
+		infoButton: 'Info',
+	},
+	totalItems = 0,
+}) => {
+	return (
+		<>
+			<ItemList expand>
+				<Item>
+					<ClayCheckbox
+						checked={checkboxChecked}
+						disabled={disabled}
+						label={
+							!filterItems && !sortingOrder && !showSearch
+								? labels.checkbox
+								: undefined
+						}
+						onChange={onCheckboxChange}
+					/>
+				</Item>
+				<Item>
+					<span className="navbar-text">
+						{selectedItems === totalItems ? (
+							labels.allSelected
+						) : (
+							<>
+								{sub(labels.selectedItemsOfTotalItems, [
+									selectedItems,
+									totalItems,
+								])}
+								<span className="navbar-breakpoint-down-d-none">
+									{labels.selected}
+								</span>
+							</>
+						)}
+					</span>
+				</Item>
+
+				{supportsBulkActions && (
+					<Item className="nav-item-shrink">
+						{clearSelectionURL ? (
+							<ClayLink
+								className="nav-link"
+								href={clearSelectionURL}
+							>
+								<span className="text-truncate-inline">
+									<span className="text-truncate">
+										{labels.clearSelection}
+									</span>
+								</span>
+							</ClayLink>
+						) : (
+							<ClayButton
+								className="nav-link"
+								displayType="unstyled"
+								onClick={onClearSelectionClick}
+								type="button"
+							>
+								<span className="text-truncate-inline">
+									<span className="text-truncate">
+										{labels.clearSelection}
+									</span>
+								</span>
+							</ClayButton>
+						)}
+					</Item>
+				)}
+
+				{showSelectAllButton && (
+					<Item className="nav-item-shrink">
+						{selectAllURL ? (
+							<ClayLink className="nav-link" href={selectAllURL}>
+								<span className="text-truncate-inline">
+									<span className="text-truncate">
+										{labels.selectAll}
+									</span>
+								</span>
+							</ClayLink>
+						) : (
+							<ClayButton
+								className="nav-link"
+								displayType="unstyled"
+								onClick={onSelectAllClick}
+								type="button"
+							>
+								<span className="text-truncate-inline">
+									<span className="text-truncate">
+										{labels.selectAll}
+									</span>
+								</span>
+							</ClayButton>
+						)}
+					</Item>
+				)}
+			</ItemList>
+
+			{(actionItems.length > 0 || showInfoButton) && (
+				<>
+					<ItemList>
+						{showInfoButton && (
```

Comment:
This will always evaluate as true.

Reply:
Because? I may only have `actionsItems` and not want to render the info button or the other way around.